### PR TITLE
Added previews for all mailers

### LINKED ISSUE #1195: Can't preview all mailers in local environment
**Describe the bug**
In development, when we go to ```localhost:8080/rails/mailers``` all mailers are not visible.

**To Reproduce**
1. Go to ```localhost:8080/rails/mailers```
2. Only few mailers are visible.

**Expected behavior**
All mailers should be visible.

**Screenshots**
Current
Screenshot from 2020-03-15 22-19-08

Expected
Screenshot from 2020-03-15 22-19-32

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by sal2701**
#### Describe the changes you have made in this PR -
- Created preview files and modified few.
- Mailers can be viewed by going to ```localhost:8080/rails/mailers```

### Screenshots of the changes (If any) -
Screenshot from 2020-03-15 22-19-32

### CODE REVIEW

File: lib/mailer_previews/assignment_mailer_preview.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+class AssignmentMailerPreview < ActionMailer::Preview
+  def new_assignment_email
+    AssignmentMailer.new_assignment_email(User.first, Assignment.first)
+  end
+
+  def update_assignment_email
+    AssignmentMailer.update_assignment_email(User.first, Assignment.first)
+  end
+
+end
```
[the remark this answers is not in the document]
Comment (sal2701) [reply lib/mailer_previews/assignment_mailer_preview.rb]: done

File: lib/mailer_previews/group_mailer_preview.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+class GroupMailerPreview < ActionMailer::Preview
+  def new_group_email
+    GroupMailer.new_group_email(User.first, Group.first)
+  end
+
+  def new_member_email
+    GroupMailer.new_member_email(User.first, Group.first)
+  end
+
+end
```
[the remark this answers is not in the document]
Comment (sal2701) [reply lib/mailer_previews/group_mailer_preview.rb]: done

File: lib/mailer_previews/pending_invitation_mailer_preview.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class PendingInvitationMailerPreview < ActionMailer::Preview
+  def new_pending_email
+    PendingInvitationMailer.new_pending_email(PendingInvitation.first)
+  end
+
```
[the remark this answers is not in the document]
Comment (sal2701) [reply lib/mailer_previews/pending_invitation_mailer_preview.rb]: done

**Comment by sal2701** [comment]
@tachyons Please review, just added this to help with viewing mailers easily. Are tests needed for these files? Coverage has decreased.

**Comment by satu0king** [comment]
Looks good to me, but who all can access `localhost:8080/rails/mailer` ? Also we wouldn't want this to be accessible in production environment.

**Comment by sal2701** [comment]
@satu0king It is accessible locally. We can't access the mails in production

**Comment by sal2701** [comment]
@satu0king @tachyons can you please merge this?

**Pull request merged**